src/constants.py

```diff
@@ -2,11 +2,18 @@
 All Constants that are used throughout the project
 """
 ADDRESS = "0x9008D19f58AAbD9eD0D60971565AA8510560ab41"
+# ADDRESS is deprecated, use SETTLEMENT_CONTRACT_ADDRESS instead
```

Comment:
Let's delete then the `ADDRESS` variable.

Reply:
I think tomorrow morning I will remove the old code for the other tests. Then this variable will be removed as well.

Reply:
Fixed.